lib/puppet/type/package.rb

```diff
@@ -384,6 +385,10 @@ def change_to_s(currentvalue, newvalue)
       desc "A read-only parameter set by the package."
     end
 
+    newparam(:package_arch) do
+      desc "Choose the architecture version of a package"
+    end
+
```

Comment:
since this is the package type, can we shorten this to `arch` or `architecture`?